common/djangoapps/student/tests/test_reset_password.py

```diff
@@ -299,3 +303,21 @@ def test_reset_password_good_token_configuration_override(self, reset_confirm):
         self.assertEquals(confirm_kwargs['extra_context']['platform_name'], 'Fake University')
         self.user = User.objects.get(pk=self.user.pk)
         self.assertTrue(self.user.is_active)
+
+    @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', "Test only valid in LMS")
+    @patch('django.core.mail.send_mail')
+    @ddt.data('Crazy Awesome Site', 'edX')
+    def test_reset_password_email_subject(self, platform_name, send_email):
+        """
+        Tests that the right platform name is included in
+        the reset password email subject
+        """
+        with patch("django.conf.settings.PLATFORM_NAME", platform_name):
+            req = self.request_factory.post(
+                '/password_reset/', {'email': self.user.email}
+            )
+            req.user = self.user
+            password_reset(req)
+            subj, _, _, _ = send_email.call_args[0]
```

Comment:
I still think that the `mail.outbox[0].subject` is cleaner, but this still works since it is consistent with the rest of the module.

I'd rather not to do any refactoring in an opensource pull request to save the taxing communication roundtrips.